src/Utils/XmlExtentions.cs

```diff
@@ -8,68 +8,24 @@ namespace Microsoft.SyndicationFeed
 {
     static class XmlExtentions
     {
-        public static ISyndicationContent ReadSyndicationContent(this XmlReader reader)
+        public static ISyndicationAttribute ReadSyndicationAttribute(this XmlReader reader)
         {
-            var content = new SyndicationContent(reader.Name, reader.NamespaceURI, null);
```

Comment:
Should this check to see if the reader is on an attribute node?

Reply:
Good catch. Adding a check